app/services/albums.js

```diff
@@ -0,0 +1,25 @@
+const request = require('request');
+const logger = require('../logger');
+const url = 'https://jsonplaceholder.typicode.com/';
+
+exports.getAlbums = (_, res1) => {
```

Comment:
Move this function to `controllers/album` file. 
Rename `res1` argument by `res`.